langchain/src/callbacks/manager.ts

```diff
@@ -432,6 +432,12 @@ export class CallbackManagerForToolRun
   }
 }
 
+function isTruthy(value: string | null | undefined): boolean {
+  if (!value) {
+    return false;
+  }
+  return /^(true|y|yes)$/i.test(value);
```

Comment:
Can we not just directly compare to `=== "true"`? Not sure I love encouraging a bunch of acceptable values here

Reply:
fair